README.md

```diff
@@ -64,23 +64,19 @@ running any other commands:
     source database/artix7/settings.sh
 
 ### Step 7: ###
-**If you are a prjxray developer, skip this step.** But, if you just want to use the Python API with a pre-generated database, just download a current stable version and skip the rest of the instructions: 
+(Option 1) - Download a current stable version (you can use the Python API with
+a pre-generated database)
 
     # WARNING: causes git issues, overrides environment
```

Comment:
This has been fixed.v can you delete this line

Reply:
Which line do you mean? The line with the "#WARNING:"?

Reply:
Yeah. If we are making this part of the main instructions now especially we should remove that